src/reducers/issues.reducer.js

```diff
@@ -1,16 +1,24 @@
 import { set } from 'lodash/fp';
 import { handleActions } from 'redux-actions';
 
+// import type BaseAction from 'types/redux.types'
+import type { Issue, Issues } from 'components/issues/issues.types';
+
 const initialState = {
   openIssues: [],
   currentIssue: {}
 };
 
+export type IssuesState = {|
+  +openIssues: Issues,
+  +currentIssue: Issue
+|};
+
 export default handleActions(
   {
-    SET_ISSUES: (state, { payload }): IssuesState =>
+    SET_ISSUES: (state: IssuesState, { payload }): IssuesState =>
```

Comment:
please do not re write the action types here. reuse the ones from the types file. 
It will prevent errors and in the case of renaming an action type you will only need to refactor one file.